packages/flutter_driver/lib/src/find.dart

```diff
@@ -26,7 +26,7 @@ abstract class CommandWithTarget extends Command {
   }
 
   /// Deserializes the command from JSON generated by [serialize].
-  CommandWithTarget.deserialize(Map<String, String> json)
```

Comment:
Makes sense. I've moved all the Command classes to be in terms of Map<String,String> as a few references were in terms of Map<String, dynamic>. I've left all responses as Map<String, dynamic> as that is the correct type for responses as per offline conversations with @yjbanov.